test/shared.py

```diff
@@ -63,3 +63,5 @@
 ALPHA_MAINNET_URL = "https://alpha-mainnet.starknet.io"
 ALPHA_GOERLI_URL = "https://alpha4.starknet.io"
 ALPHA_GOERLI2_URL = "https://alpha4-2.starknet.io"
+
+MAX_FEE = int(4e16)
```

Comment:
Rather than defining it here, I'd rather it's defined in each test.

Reply:
Removed in latest update